src/components/RichTextEditor/context/RichTextEditorContext.tsx

```diff
@@ -10,28 +10,28 @@ import { EditorResizeContextProvider } from './EditorResizeContext';
 export type RichTextEditorContextProps = {
     designTokens: DesignTokens;
     position: Position;
+    border: boolean;
 };
 
 const RichTextEditorContext = createContext<RichTextEditorContextProps>({
     designTokens: defaultDesignTokens,
     position: Position.FLOATING,
```

Comment:
Here you need to adjust the right return from context.

Reply:
fixed